src/Quiz/Quiz.js

```diff
@@ -80,6 +80,8 @@ const Quiz = (props) => {
     setError(false)
     const submissionData = compileSubmissionData()
     getPackingListData(submissionData)
+    .then(data => console.log(data))
```

Comment:
Should we remove this console log and the extra line?

Reply:
Yes! We should also change the catch error to `(() => console.error)`